recipe/build.sh

```diff
@@ -1,10 +1,15 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
 pushd tesseract
 
-autoreconf -fi
-LIBLEPT_HEADERSDIR=$PREFIX/include ./configure --prefix=$PREFIX --with-extra-libraries=$PREFIX/lib
-LDFLAGS="-L$PREFIX/lib" CFLAGS="-I$PREFIX/include" make -j $CPU_COUNT
+./autogen.sh
+LIBLEPT_HEADERSDIR="${PREFIX}/include" ./configure \
+    --prefix="${PREFIX}" \
+    --with-extra-libraries="${PREFIX}/lib" \
+    CC="${CC}" \
+    CXX="${CXX}" \
+    || { cat config.log; exit 1; }
+make -j $CPU_COUNT
```

Comment:
Thanks for tidying this a bit. Nice to see the `LDFLAGS` and `CFLAGS` are getting dropped. Means the compilers are doing their job. :)